libvirt.go

```diff
@@ -257,13 +258,17 @@ func (d *Driver) Create() error {
 	}
 
 	config := DomainConfig{
-		DomainName: d.MachineName,
-		Memory:     d.Memory,
-		CPU:        d.CPU,
-		CacheMode:  d.CacheMode,
-		IOMode:     d.IOMode,
-		DiskPath:   d.DiskPath,
-		Network:    d.Network,
+		DomainName:   d.MachineName,
+		Memory:       d.Memory,
+		CPU:          d.CPU,
+		CacheMode:    d.CacheMode,
+		IOMode:       d.IOMode,
+		DiskPath:     d.DiskPath,
+		Network:      d.Network,
+		ExtraDevices: []string{},
```

Comment:
Isn't this going to be empty by default if you omit it?

Reply:
done :+1: